AnkiDroid/src/main/java/com/ichi2/libanki/Collection.java

```diff
@@ -2103,7 +2107,23 @@ public Media getMedia() {
     }
 
 
-    public Models getModels() {
+    /**
+     * On first call, load the model if it was not loaded.
+     *
+     * Synchronized to ensure that loading does not occur twice.
+     * Normally the first call occurs in background when collection
+     * start.  The only exception being if the user request an action
+     * (e.g. review) so quickly after start that
+     * loadModelsInBackground had no time to be called. In this case,
```

Comment:
Nit/implementer's choice:
Best to keep documentation a little less verbose or people will skim over it.

```suggestion
     * loadModelsInBackground had no time to be called. In this case, it will instantly finish.
```

Reply:
Done